### PR TITLE
MAT-8343 We are removing errorType field and setting it to null, so f…

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-8343](https://jira.cms.gov/browse/MAT-8343)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by mcmcphillips — DISMISSED** [review]

File: src/main/java/gov/cms/mat/cql_elm_translation/controllers/CqlConversionController.java [hunk 1]
```diff
@@ -77,7 +76,7 @@ public CqlConversionPayload cqlToElmJson(
             .build();
     cqlLibraryService.setUpLibrarySourceProvider(cqlData, accessToken);
 
-    return cqlConversionService.translateCqlToElm(requestData, checkContext);
+    return cqlConversionService.translateCqlToElm(requestData);
```
Comment (adongare) [inline src/main/java/gov/cms/mat/cql_elm_translation/controllers/CqlConversionController.java]: is this not required anymore? this might cause an issue for libraries. I think measure needs to have context but not for standalone library

Comment (RohitKandimalla) [reply src/main/java/gov/cms/mat/cql_elm_translation/controllers/CqlConversionController.java]: Reverted

**Review by adongare — APPROVED** [review]
